paper/paper.md

```diff
@@ -0,0 +1,152 @@
+users the ability to combine different types of regularization with different noise
+distributions in the GLMs.
+
+Despite the attractiveness of regularized GLMs, the available tools in
+the Python data science eco-system are highly fragmented. More
+specifically,
+
+-  [statsmodels] provides a wide range of link functions but no regularization.
+-  [scikit-learn] provides elastic net regularization but only for linear models.
+-  [lightning] provides elastic net and group lasso regularization, but only for
+   linear and logistic regression.
+
+[Pyglmnet] is a response to this fragmentation. Here are is a comparison of Pyglmnet with existing toolboxes.
+
+|                    | [pyglmnet] | [scikit-learn] | [statsmodels] |   [lightning]   |   [py-glm]    | [Matlab]|   [glmnet] in R |
+|--------------------|:----------:|:--------------:|:-------------:|:---------------:|:-------------:|:-------:|:---------------:|
+| **distributions**  |            |                |               |                 |               |         |                 |
+| gaussian           |    x       |      x         |      x        |       x         |      x        |    x    |  x              |
+| binomial           |    x       |      x         |      x        |       x         |      x        |    x    |  x              |
+| poisson            |    x       |                |      x        |                 |      x        |    x    |  x              |
+| softplus           |    x       |                |               |                 |               |         |                 |
+| probit             |    x       |                |               |                 |               |         |                 |
+| gamma              |    x       |                |               |                 |               |    x    |                 |
+| **regularization** |            |                |               |                 |               |         |                 |
+| l2                 |    x       |      x         |               |       x         |               |         |                 |
+| lasso              |    x       |      x         |               |       x         |               |         |  x              |
+| group lasso        |    x       |                |               |       x         |               |         |  x              |
+| tikhonov           |    x       |                |               |                 |               |         |                 |
+
+Pyglmnet implements the same algorithm described in [Friedman, J., Hastie, T., & Tibshirani, R. (2010)](https://core.ac.uk/download/files/153/6287975.pdf>) and the accompanying widely popular R package [glmnet].
+
+As opposed to other implementations which are wrappers around this package, it is written in pure Python and runs on Python 3.5+. The implementation is compatible with the existing data science ecosystem.
```

Comment:
In this para "other implementations" seems to refer to `scikit-learn` / `statsmodels` / `lightning` but we are really referring to `glmnet-python` from Civis Analytics and other similar recent implementations. Can we make that clearer?